save-core/src/nativeTest/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternalTest.kt

```diff
@@ -10,12 +10,12 @@ class ProcessBuilderInternalTest {
     @Test
     fun `check stdout with redirection`() {
         val actualResult = processBuilder.exec("echo something >/dev/null", null)
-        val expectedCode = 0
-        val expectedStdout = "something"
+        val expectedCode = if (isCurrentOsWindows()) 1 else 0
```

Comment:
Why does it return 1 now? Wouldn't it be confusing?

Reply:
You can look at [this](https://stackoverflow.com/questions/7105433/windows-batch-echo-without-new-line/11336754#11336754)

For now, exit codes are checked only by our tests, also I I don't think, that users will use `echo` commands regularly in SAVE. It's a good question, but I don't think, that to use some dummName (advise  in stackoverflow) is good approach too, so I left it as-is